### PR TITLE
[frax-finance] Frax Finance: Two different voting strategies

### PR DESCRIPTION
**Pull request by FortisFortuna**
Fixes previous Frax issues. Gives two different voting strategies

### CODE REVIEW

File: src/strategies/frax-finance/index.ts [hunk 1]
```diff
@@ -5,13 +5,11 @@ import { multicall } from '../../utils';
 const BIG6 = BigNumber.from('1000000');
 const BIG18 = BigNumber.from('1000000000000000000');
 
-const UNISWAP_SUBGRAPH_URL = {
-  '1': 'https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v2'
-};
-
 export const author = 'FraxFinance';
 export const version = '0.0.1';
```
Comment (ChaituVR) [inline src/strategies/frax-finance/index.ts:9]: would be good to raise this version so others can understand there is a change, also if you have some time would be nice to have `frax-finance` strategy changes in a separate PR

Comment (FortisFortuna) [reply src/strategies/frax-finance/index.ts]: I consolidated into one strategy in 986a603

**Comment by jasonhuan** [comment]
hey! just wondering if we could get a review soon, since we're waiting to post a governance proposal asap

File: src/strategies/frax-finance/index.ts [hunk 2]
```diff
@@ -198,125 +168,59 @@ export async function strategy(
...
-        // console.log("Free Sushi FRAX/FXS LP: ", free_sushi_frax_fxs.div(BIG18).toString());
-        // console.log("Farmed Sushi FRAX/FXS LP [boosted]: ", farm_sushi_frax_fxs.div(BIG18).toString());
-        // console.log("Free Sushi FXS/WETH: ", free_sushi_fxs_weth.div(BIG18).toString());
-        // console.log("Farmed Sushi FXS/WETH [boosted]: ", farm_sushi_fxs_weth.div(BIG18).toString());
-        // console.log("------");
-        // console.log("FXS per Uni FRAX/FXS LP: ", uniLPFraxFxs_fxs_per_LP_E18.toString());
-        // console.log("FXS per Sushi FRAX/FXS LP: ", sushiLPFraxFxs_fxs_per_LP_E18.toString());
-        // console.log("FXS per Sushi FXS/WETH LP: ", sushiLPFxsWeth_fxs_per_LP_E18.toString());
-        // console.log(``);
+        console.log(`==================${addresses[i]}==================`);
```
Comment (bonustrack) [inline src/strategies/frax-finance/index.ts:199]: Can you remove the console.log?

Comment (jasonhuan) [reply src/strategies/frax-finance/index.ts]: done

**Review by bonustrack — APPROVED** [review]

**Pull request merged**